src/main/resources/templates/goals/create.html

```diff
@@ -1,65 +1,79 @@
-                    <small id="goalHelp" class="form-text text-muted">What do you want to accomplish?</small>
+                    <input aria-describedby="goalHelp" class="form-control" id="goal" name="goal" type="text">
+                    <small class="form-text text-muted" id="goalHelp">What do you want to accomplish?</small>
                 </div>
                 <div class="form-group">
                     <label for="description">Description</label>
-                    <textarea type="text" class="form-control" id="description" name="description"></textarea>
+                    <textarea class="form-control" id="description" name="description" type="text"></textarea>
                 </div>
                 <div class="form-group">
                     <label for="progress">Progress</label>
-                    <input type="number" class="form-control" id="progress" name="progress"
-                           aria-describedby="progressHelp">
-                    <small id="progressHelp" class="form-text text-muted">How far have you gotten so far?</small>
+                    <input aria-describedby="progressHelp" class="form-control" id="progress" name="progress"
+                           type="number">
+                    <small class="form-text text-muted" id="progressHelp">How far have you gotten so far?</small>
                 </div>
                 <div class="form-group">
                     <label for="target">Target</label>
-                    <input type="number" class="form-control" id="target" name="target" aria-describedby="targetHelp">
-                    <small id="targetHelp" class="form-text text-muted">At what point is this goal accomplished?</small>
+                    <input aria-describedby="targetHelp" class="form-control" id="target" name="target" type="number">
+                    <small class="form-text text-muted" id="targetHelp">At what point is this goal accomplished?</small>
                 </div>
                 <div class="form-group">
                     <label for="dueDate">Due Date</label>
                     <!-- placeholder workaround for Safari :( shouldn't affect other browsers-->
-                    <input type="date" placeholder="yyyy-mm-dd" class="form-control" id="dueDate" name="due_date"
-                           aria-describedby="dueDateHelp">
-                    <small id="dueDateHelp" class="form-text text-muted">Does this goal have a due date?</small>
+                    <input aria-describedby="dueDateHelp" class="form-control" id="dueDate" name="due_date"
+                           placeholder="yyyy-mm-dd"
+                           type="date">
+                    <small class="form-text text-muted" id="dueDateHelp">Does this goal have a due date?</small>
+                </div>
+                <div class="form-group">
+                    <label for="recurrence">Recurrence</label>
+                    <p>
```

Comment:
This field doesn't do anything? Shouldn't this type of change be a part of a commit that utilizes this value in the system?

Reply:
Not necessarily. 

Also, the field creates a new line.

Reply:
Alrighty, I don't think it breaks anything, so works out